### ISSUE TITLE
[Bug]: Pressing Control-Option-Space in Safari when using Voiceover does not open the menu

### ISSUE THREAD

**Issue by imgiseverything** [add2cal/add-to-calendar-button#379]
### What happened?

When the button is the document's 'activeElement' e.g. it has focus, the button should open the menu iwhen the user presses the ENTER key or the SPACE key. 

Also, when using the Voiceover screen-reader on MacOS, the user should be able to press the Control+Option+Space combination too. This combination is announced to the user so they would expect it to work.

It seems the SPACE key does not work either.

### How to reproduce?

Open up the Voiceover screen-reader on an Apple MacOS computer.

Open Safari and go to https://add-to-calendar-button.com/#demo, press the TAB key until you reach the 'Add to Calendar' button. 

Voiceover will announce that you can press Control-Option-Space to action the button but on doing so nothing happens

### Screenshots & more

![image](https://user-images.githubusercontent.com/221242/234274304-c14173b4-fee5-4038-a1c6-23059e2d4ed8.png)

### System & Environment

1. Macbook M2 Pro - MacOS  13.2.1
2. Safari 16.3
3. Whatever is used on the demo site :)

**Comment by jekuer** [comment]
Thanks!
Will become part of the next minor release.
